config/config.go

```diff
@@ -81,7 +81,7 @@ type configData struct {
 	Debug               bool                  `json:"debug,omitempty"`
 	DisablePartialStart bool                  `json:"disable_partial_start"`
 	EnableWebProfile    bool                  `json:"enable_web_profile"`
-	LogConfig           []LoggerPatternConfig `json:"log"`
+	LogConfig           []LoggerPatternConfig `json:"log,omitempty"`
```

Comment:
Do we not need a `LogConfig` field on the `Config` struct above, too?

Reply:
There is a field [here](https://github.com/viamrobotics/rdk/blob/3b8b07f972a2810b246e9c4e4c94c9247adeba86/config/config.go#L40). Should I move it to the bottom of the struct? I see why it is hard to find

Reply:
It's already added [on line 40](https://github.com/viamrobotics/rdk/pull/4193/files/c5eb8f68da3d55a5819eeae54e222ba9c631951e#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R40), but maybe it's worth adding a comment for it.

Reply:
Ah thanks for pointing it out; nope all good, you can leave where is. Just missed it 😮‍💨 .